src/main/steps/c100-rebuild/safety-concerns/concern-guidance/content.test.ts

```diff
@@ -73,44 +73,46 @@ const cy = {
   abuseList: [
-    'physical or sexual - welsh',
-    'psychological - welsh',
-    'emotional - welsh',
-    'violent or threatening behaviour - welsh',
-    'controlling or coercive behaviour - welsh',
-    'economic, meaning that they limit your ability to acquire, use or maintain money or other property, or acquire goods or services. - welsh',
+    'gorfforol neu rywiol',
+    'seicolegol',
+    'emosiynol',
+    'ymddygiad treisgar neu fygythiol',
+    'ymddygiad rheolaethol neu gymhellol',
+    "economaidd, sy'n golygu eu bod yn cyfyngu ar eich gallu i ennill, defnyddio neu gadw arian neu eiddo arall, neu derbyn nwyddau neu wasanaethau",
   ],
-  otherFormsLine: 'It could also take other forms, such as abducting the children. - welsh',
-  abusiveGuidance: 'If you are not sure if their behaviour is abusive, see the guidance on - welsh',
-  andSignsOf: 'and the signs of - welsh',
+  otherFormsLine: 'Gallai hefyd gymryd ffurfiau eraill, fel cipio plant.',
+  abusiveGuidance: 'Os nad ydych yn siŵr a yw eu hymddygiad yn ddifrïol, edrychwch ar y canllawiau ar',
+  andSignsOf: 'ac arwyddion o',
   typesOfAbusiveBehaviour:
-    '<a href="https://supportnav.org.uk/what-is-domestic-abuse" class="govuk-link" target="_blank" aria-label="types of abusive behaviour - welsh">types of abusive behaviour - welsh</a>',
+    '<a href="https://supportnav.org.uk/what-is-domestic-abuse" class="govuk-link" target="_blank" aria-label="types of abusive behaviour - welsh">fathau o ymddygiad difrïol</a>',
   childAbuse:
-    '<a href="https://www.nspcc.org.uk/what-is-child-abuse/types-of-abuse/" class="govuk-link" target="_blank" aria-label="child abuse. - welsh">child abuse. - welsh</a>',
-  infoUsedByCourtHeader: 'How the court will use this information - welsh',
+    '<a href="https://www.nspcc.org.uk/what-is-child-abuse/types-of-abuse/" class="govuk-link" target="_blank" aria-label="child abuse. - welsh">gam-drin plant</a>',
+  infoUsedByCourtHeader: "Sut bydd y llys yn defnyddio'r wybodaeth hon",
   infoList: [
-    'The court needs to know about this behaviour, to make sure that any orders are in the best interests of you and the children. - welsh',
-    'The court will use the information you provide to handle your case correctly. - welsh',
+    "Mae’r llys angen gwybod am yr ymddygiad hwn, i wneud yn siŵr bod unrhyw orchmynion er eich lles chi a'r plant.",
+    "Bydd y llys yn defnyddio'r wybodaeth y byddwch yn ei darparu i drin eich achos yn gywir",
   ],
   warningText: {
-    text: 'The information that you give in this section will also be shared with the other people in this application, so they are able to respond to what you have said. - welsh',
-    iconFallbackText: 'Warning - welsh',
+    text: "Bydd yr wybodaeth y byddwch yn ei rhoi yn yr adran hon hefyd yn cael ei rhannu â'r bobl eraill yn y cais hwn, fel eu bod nhw yn gallu ymateb i'r hyn yr ydych chi wedi'i ddweud.",
+    iconFallbackText: 'Rhybudd',
```

Comment:
@sealDebolina  This is not Warning class to make it as warning banner.
We should not update it with welsh, please check it once if this works fine ?

Reply:
This applies to all warning texts we have used in the c100

Reply:
This is done.
https://github.com/hmcts/prl-citizen-frontend/pull/704/commits/0545aaa110180faae064df6827ca03630fc8b4cf